### PR TITLE
Test installer commands (retry)

### PR DESCRIPTION
**Pull request by jonathanhefner**
This is a retry of #182.  I have added a gemfile for Rails 6.1, and refreshed all gemfile lock files.

@dhh I did not track down the root cause of the messages in https://github.com/rails/jsbundling-rails/pull/196#issue-2436103482, but they only appear when testing with Rails 6.0.  Currently, `main` branch [tests with Rail 6.1 by default](https://github.com/rails/jsbundling-rails/blob/6ead108f3670b4fd937f38296fdb0a689c7d5df9/Gemfile#L7).  In #182, I had changed that to Rails 6.0 to [match the gemspec](https://github.com/rails/jsbundling-rails/blob/44919fe2ecc11fd23b26fc39692615d87656da7f/jsbundling-rails.gemspec#L14), but I have changed it back to Rails 6.1 in this PR.

In summary, after this PR is merged: the gemspec will require Rails >= 6.0, `bundle exec rake` will test with Rails 6.1, and CI will test with Rails >= 6.1.

### CODE REVIEW

**Comment by dhh** [comment]
That worked!

**Comment by jonathanhefner** [comment]
@dhh

> Could you add a bin/test binstub as well to trigger this?

Do you mean a binstub to run `bundle exec rake` with just the default `Gemfile`, or once for each gemfile?  I've submitted #198 to do the latter, but I can strip out the loop if you prefer.

> And maybe we should also get some github hooks setup to run automatically on PRs.

The GitHub workflow from this PR will trigger on PRs:

https://github.com/rails/jsbundling-rails/blob/2413e21cd6d6c42bbaad1d3275b50faac63707ba/.github/workflows/ci.yml#L2

Or did you mean something else?

**Comment by dhh** [comment]
Was thinking that bin/test should run everything.

Ahh, hadn't noticed that on the ci.yml. All good then.
